Makefile

```diff
@@ -5,4 +5,14 @@ SOURCE   = https://github.com/ofiwg/$(NAME)/archive/v$(VERSION).tar.$(SRC_EXT)
 sle12_REPOS := --repo https://download.opensuse.org/repositories/science:/HPC/openSUSE_Leap_42.3/
 sl42_REPOS  := --repo https://download.opensuse.org/repositories/science:/HPC/openSUSE_Leap_42.3/
 
+ifneq ($(DAOS_STACK_REPO_SUPPORT),"")
+ifneq ($(DAOS_STACK_REPO_UBUNTU_18_04_LIST),"")
+ubuntu1804_REPOS := $(shell curl $(DAOS_STACK_REPO_SUPPORT)$(DAOS_STACK_REPO_UBUNTU_18_04_LIST))
```

Comment:
What's the benefit/requirement of fetching these rather than just specifying them here?

Reply:
The variable needs the contents of the file(s) in order to pass it to pbuilder as additional repositories.
The environment variables allow a site to customize where they have the file located.

Reply:
Yeah.  I'm just trying to determine why the content of this needs to be fetched rather than just be here directly.